tests/phpunit/tests/theme/addEditorStyle.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * @group themes
+ * @ticket 52294
+ *
+ * @covers ::add_editor_style
+ */
+
+class Test_Add_Editor_Style extends WP_UnitTestCase {
+	/**
+	 * Reset the editor styles to an empty array.
+	 */
+	public function set_up() {
+		parent::set_up();
+		global $editor_styles;
+		$editor_styles = array();
+	}
+
+	/**
+	 * Tests the default editor styles in RTL mode.
+	 *
+	 * @return void
+	 */
+	public function test_add_editor_style_default() {
+		global $editor_styles;
+
+		add_editor_style();
+		$this->assertSame( array( 'editor-style.css' ), $editor_styles );
+	}
+
+	/**
+	 * run default setting in rtl mode.
```

Comment:
Docblock summaries should begin with a capital letter and use third-person singular verbs (i.e. "Runs", not "run").

🔢 Applies to other instances in the PR.